backend/app/api/src/endpoints/organization/webshops/RetrieveUitpasSocialTariffPriceEndpoint.ts

```diff
@@ -26,26 +27,40 @@ export class RetrieveUitpasSocialTariffPricesEndpoint extends Endpoint<Params, Q
     }
 
     async handle(request: DecodedRequest<Params, Query, Body>) {
-        if (request.body.uitpasEventId) {
+        if (request.body.uitpasEventUrl) {
             // OFFICIAL FLOW
             if (!request.body.uitpasNumbers) {
                 // STATIC CHECK
                 // request shouldn't include a reduced price
+
+                // this call should be authenticated, as it is done from the webshop settings
+                const organization = await Context.setOrganizationScope({ willAuthenticate: true });
+                await Context.authenticate();
+                const reducedPrice = await UitpasService.getSocialTariffForEvent(
+                    organization.id,
+                    request.body.basePrice, // Convert from cents to euros
```

Comment:
Hier staat iets over convert to euro en omgekeerd, maar dat zie ik niet in de code? Misschien nog een oude comment